src/azle.ts

```diff
@@ -472,7 +472,7 @@ function runAzleGenerate(
         );
 
         return Err({
-            error: `${generalErrorMessage}\n\n${errorLines.join('\n')}`,
+            error: errorLines.join('\n'),
```

Comment:
This removes the "Something about your typescript violates Azle's requirements message in most instances.